packages/headless-react/tsconfig.test.json

```diff
@@ -1,27 +1,10 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "./tsconfig.json",
   "compilerOptions": {
-    "allowUnreachableCode": false,
-    "allowSyntheticDefaultImports": true,
-    "allowJs": true,
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "esModuleInterop": true,
-    "noImplicitAny": true,
-    "noImplicitReturns": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "removeComments": false,
-    "sourceMap": true,
-    "declaration": true,
-    "jsx": "react-jsx",
-    "outDir": "dist/",
+    // Note: ESM config based on following official docs doesn't work
+    //  https://jestjs.io/docs/ecmascript-modules
+    //  https://kulshekhar.github.io/ts-jest/docs/guides/esm-support/
```

Comment:
Hmmm check: github.com/coveo/semantic-monorepo-tools, iirc it works A-1 there and its ESM